src/codegen/EndpointCodeFile.ts

```diff
@@ -11,6 +11,19 @@ export const endpointCodeGenerator = generatorFactory<IEndpointDefinition>(
     const bodyArg = (this.def.args ?? []).find(a => a.paramType.type === "body");
     const bodyArgContentType = getContentType(bodyArg?.type);
 
+    const queryParams = (this.def.args ?? []).filter(isQueryArgument);
+    const queryArg = queryParams.length === 0 ? undefined : `{ ${
```

Comment:
Code gen now checks for supported types but otherwise just passes the query params to `conjureFetch`.